mods/ra/maps/monster-tank-madness/weapons.yaml

```diff
@@ -7,28 +7,12 @@ TurretGun:
 		Blockable: false
 
 SuperTankPrimary:
-	Inherits: ^Cannon
-	ROF: 70
+	Inherits: 120mm
+	ReloadDelay: 70
 	Range: 4c768
```

Comment:
Minor nit: This is also inherited from 120mm (or rather ^Cannon), so it could be removed as well.